hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestWritableECContainerProvider.java

```diff
@@ -234,9 +234,11 @@ public Pipeline createPipeline(ReplicationConfig repConf,
     provider = new WritableECContainerProvider(
         conf, pipelineManager, containerManager, pipelineChoosingPolicy);
 
-    ContainerInfo container =
-        provider.getContainer(1, repConfig, OWNER, new ExcludeList());
-    assertNull(container);
+    try {
+      provider.getContainer(1, repConfig, OWNER, new ExcludeList());
```

Comment:
You have to add Assert.fail here. Lets say if above statement does not throw exception, then case will pass.

Reply:
Thank for pointing it out. Updated the PR.